### ISSUE TITLE
Update Cargo deps for Pear/Rocket Rust Nightly regression workaround

### ISSUE THREAD

**Issue by rayrrr** [agersant/polaris#84]
Hello again. I recently had to start from scratch with Polaris again and it wouldn't build on Linux this time around even though it had built successfully on the same hardware a couple months back. Here's why:

https://github.com/rust-lang/rust/pull/73293

This change to Rust created a regression in Pear (a Rocket subdependency), but thankfully a bugfix has been released. After following the directions in the above PR to take advantage of the bugfix, the build succeeded.

Basically, we need to update Pear to its latest version in the `Cargo.lock` file for Polaris to successfully build with Rust Nightly.

Something like `cargo update -p pear && cargo update` should do it, then we can put up a PR with the Cargo file changes.

Would you like to handle this, or for me to put up a PR?
